From: Ben Hutchings <ben@decadent.org.uk>
Date: Mon, 29 Oct 2018 00:14:06 +0000
Subject: Revert "inet: frags: reorganize struct netns_frags"
Forwarded: not-needed

This reverts commit 5b68fda0a455be7f48fdf97407de1aa09d046fdd, which
causes an ABI change.
---
 include/net/inet_frag.h | 9 ++++-----
 1 file changed, 4 insertions(+), 5 deletions(-)

--- a/include/net/inet_frag.h
+++ b/include/net/inet_frag.h
@@ -4,17 +4,16 @@
 #include <linux/rhashtable.h>
 
 struct netns_frags {
+	struct rhashtable       rhashtable ____cacheline_aligned_in_smp;
+
+	/* Keep atomic mem on separate cachelines in structs that include it */
+	atomic_long_t		mem ____cacheline_aligned_in_smp;
 	/* sysctls */
 	long			high_thresh;
 	long			low_thresh;
 	int			timeout;
 	int			max_dist;
 	struct inet_frags	*f;
-
-	struct rhashtable       rhashtable ____cacheline_aligned_in_smp;
-
-	/* Keep atomic mem on separate cachelines in structs that include it */
-	atomic_long_t		mem ____cacheline_aligned_in_smp;
 };
 
 /**
